js/stateless.js/src/rpc-interface.ts

```diff
@@ -68,21 +77,21 @@ const BN254FromString = coerce(instance(BN), string(), (value) =>
  */
 const Base64EncodedUtxoDataResult = tuple([string(), literal('base64')]);
 
-/**
- * @internal
- */
-const TlvFromBase64EncodedUtxoData = coerce(
-  instance(Array<TlvDataElement_IdlType>),
-  Base64EncodedUtxoDataResult,
-  (value) => {
-    const decodedData = decodeUtxoData(Buffer.from(value[0], 'base64'));
-    if (decodedData.tlvElements.every(isValidTlvDataElement)) {
-      return decodedData;
-    } else {
-      throw new Error('Invalid TlvDataElement structure');
-    }
-  },
-);
+// /**
+//  * @internal
+//  */
+// const TlvFromBase64EncodedUtxoData = coerce(
+//     instance(Array<CompressedAccountData>),
+//     Base64EncodedUtxoDataResult,
+//     value => {
+//         const decodedData = decodeUtxoData(Buffer.from(value[0], 'base64'));
+//         if (decodedData.tlvElements.every(isValidTlvDataElement)) {
+//             return decodedData;
+//         } else {
+//             throw new Error('Invalid TlvDataElement structure');
+//         }
+//     },
+// );
```

Comment:
leftover comment?

Reply:
and other commented code in this file

Reply:
will be needed, i will sync rpc-interface with photon in a different pr